src/WCSimPMTObject.cc

```diff
@@ -212,14 +212,24 @@ G4float* PMT20inch::GetQEWavelength(){
 }
 
 G4float* PMT20inch::GetQE(){  
-  static G4float QE[20] = { 0.00, .0139, .0854, .169, .203, .206, .211, .202,.188, .167, .140, .116, .0806, .0432, .0265, .0146, .00756, .00508, .00158, 0.00};
+  G4float correctionFactor = 1./0.73;//Correction factor added in July 2015 to set R3600 C.E. to 73% and not to change the R3600 output.
+  static G4float QE[20] = { 0.00*correctionFactor, .0139*correctionFactor, .0854*correctionFactor, .169*correctionFactor, .203*correctionFactor, .206*correctionFactor, .211*correctionFactor, .202*correctionFactor,.188*correctionFactor, .167*correctionFactor, .140*correctionFactor, .116*correctionFactor, .0806*correctionFactor, .0432*correctionFactor, .0265*correctionFactor, .0146*correctionFactor, .00756*correctionFactor, .00508*correctionFactor, .00158*correctionFactor, 0.00*correctionFactor};
+  //  static G4float QE[20] = { 0.00, .0139, .0854, .169, .203, .206, .211, .202,.188, .167, .140, .116, .0806, .0432, .0265, .0146, .00756, .00508, .00158, 0.00};
+
+
   return QE;
```

Comment:
If we are going to do it this way, you should just return QE*correctionFactor rather than putting it in the array.